src/containers/collectibles/components/CollectiblesPage.tsx

```diff
@@ -0,0 +1,629 @@
+        headerContainerClassName={styles.modalHeader}
+        titleClassName={styles.modalTitle}
+        allowScroll
+      >
+        <div className={styles.editModal}>
+          <div className={styles.editListSection}>
+            <DragDropContext onDragEnd={onDragEnd}>
+              <div className={styles.editListHeader}>Visible Collectibles</div>
+
+              <div className={styles.editTableContainer}>
+                <Droppable droppableId={VISIBLE_COLLECTIBLES_DROPPABLE_ID}>
+                  {(provided, snapshot) => (
+                    <div ref={provided.innerRef} {...provided.droppableProps}>
+                      {getVisibleCollectibles().map((c, index) => (
+                        <Draggable key={c.id} draggableId={c.id} index={index}>
+                          {(provided, snapshot) => (
+                            <VisibleCollectible
+                              {...provided.draggableProps}
+                              handleProps={provided.dragHandleProps}
+                              forwardRef={provided.innerRef}
+                              name={c.name}
+                              imageUrl={c.imageUrl}
+                              isOwned={c.isOwned}
+                              dateCreated={c.dateCreated}
+                              onHideClick={handleHideCollectible(c.id)}
+                            />
+                          )}
+                        </Draggable>
+                      ))}
+
+                      {provided.placeholder}
+                    </div>
+                  )}
+                </Droppable>
+              </div>
+            </DragDropContext>
+          </div>
+
+          <div className={styles.editListSection}>
+            <div className={styles.editListHeader}>Hidden Collectibles</div>
+
+            <div className={styles.editTableContainer}>
+              {getHiddenCollectibles().map(c => (
+                <HiddenCollectible
+                  key={c.id}
+                  name={c.name!}
+                  imageUrl={c.imageUrl!}
+                  isOwned={c.isOwned}
+                  dateCreated={c.dateCreated}
+                  onShowClick={handleShowCollectible(c.id)}
+                />
+              ))}
+            </div>
+          </div>
+
+          <Button
+            className={styles.editDoneButton}
+            type={ButtonType.PRIMARY_ALT}
+            size={ButtonSize.SMALL}
+            text='Done'
+            onClick={handleDoneClick}
+          />
+        </div>
+      </Modal>
+
+      <Drawer
+        isOpen={showUseDesktopDrawer}
+        onClose={() => setShowUseDesktopDrawer(false)}
+      >
+        <div className={styles.editDrawer}>
+          <IconGradientCollectibles className={styles.editDrawerIcon} />
+          <div className={styles.editDrawerTitle}>Edit Collectibles</div>
+          <div className={styles.editDrawerContent}>
+            Visit audius.co from a desktop browser to hide and sort your NFT
```

Comment:
could you put this text in the messages var

Reply:
yes sir!